cli/cmd/org/edit.go

```diff
@@ -12,6 +12,7 @@ import (
 
 func EditCmd(ch *cmdutil.Helper) *cobra.Command {
 	var orgName, displayName, description, billingEmail string
+	var flagsSet bool
```

Comment:
This variable is only used inside `RunE` so should not be declared in the outer scope (which makes it look like a flag value). E.g. around line 48 seems like a more suitable place.

This applies in several files.